### PR TITLE
Schedule ReplicationControllers in pod-startup-time test.

### PR DESCRIPTION
**Pull request by wojtek-t**
@derekwaynecarr - FYI

### CODE REVIEW

File: test/e2e/density.go [hunk 1]
```diff
@@ -552,40 +549,51 @@ var _ = Describe("Density", func() {
 			By("Removing additional pods if any")
```
Comment (gmarek) [inline test/e2e/density.go]: s/pods/rcs/

File: test/e2e/density.go [hunk 2]
```diff
@@ -552,40 +549,51 @@ var _ = Describe("Density", func() {
...
 				name := additionalPodsPrefix + "-" + strconv.Itoa(i)
-				c.Pods(ns).Delete(name, nil)
+				c.ReplicationControllers(ns).Delete(name)
 			}
 		})
 	}
 })
 
-func createRunningPod(wg *sync.WaitGroup, c *client.Client, name, ns, image string, labels map[string]string, cpuRequest, memRequest resource.Quantity) {
+func createRunningPod(wg *sync.WaitGroup, c *client.Client, name, ns, image, podType string, cpuRequest, memRequest resource.Quantity) {
```
Comment (gmarek) [inline test/e2e/density.go]: s/createRunningPod/createRunningRC/

**Comment by gmarek** [comment]
Just 2 nits.

File: test/e2e/density.go (hunk 1 above)
Comment (wojtek-t) [inline test/e2e/density.go]: done

File: test/e2e/density.go (hunk 2 above)
Comment (wojtek-t) [inline test/e2e/density.go]: done

**Comment by wojtek-t** [comment]
@gmarek - PTAL

**Pull request merged**